include/arch/arm/aarch64/scripts/linker.ld

```diff
@@ -39,6 +39,8 @@
   #define ROM_ADDR (CONFIG_FLASH_BASE_ADDRESS + CONFIG_FLASH_LOAD_OFFSET)
 #endif
 
+#define PAGE_SIZE	0x1000
```

Comment:
Can you document this value? Is this arbitrary selection?

Reply:
While AArch64 supports multiple page sizes, this PR only supports 4KiB (which is quite reasonable; e.g. x86 uses 4KiB as default as well).

Reply:
OK. Still, needs inline explanation

Reply:
Will add inline explanation.